lib/features/mailbox/presentation/mailbox_controller.dart

```diff
@@ -283,7 +283,7 @@ class MailboxController extends BaseMailboxController with MailboxActionHandlerM
         mailboxDashBoardController.clearMailboxUIAction();
       } else if (action is RefreshChangeMailboxAction) {
         if (action.newState != currentMailboxState) {
-          _refreshMailboxChanges();
+          _refreshMailboxChanges(properties: action.properties);
```

Comment:
good catch, maybe resolve many issues :))

Reply:
I don't think we need this.